README.md

```diff
@@ -240,6 +240,10 @@ Send unhandled errors and performance metrics to sentry.
 
 Configure Sentry DSN using `[p]set api sentry dsn,https://fooo@bar.baz/9`, then load the Cog `[p]load sentry`.
 
+Sentry environment can be configured using `[p]sentry_set_env my-discord-server`.
+
+Test the cog by running `[p]sentry_test`.
```

Comment:
What does this actually do? Is it an output test or a Sentry server communication test?

Reply:
Basically, in sentry you have environments which allows you to separate issues and performance into different groups. With this setting, you'd set an environment for every guild where the bot is added, and then those metrics are separated based on that.

I thought about just using the guild name as environment so make it even easier to configure, however that may change from time to time so I'd rather have a static config entry.

Reply:
My bad, I didn't realise this highlighted 3 lines. I'm talking about line 245.

Reply:
Oh, 245 just raises an Exception that is sent to sentry (only mods can execute the command), so that you can see if everything works correctly.

Reply:
Gotcha, OK.
```suggestion
The configuration can be tested by sending a test exceptiion to Sentry using `[p]sentry_test`.
```